### DISCUSSION TITLE
建议创建一个ppa仓库方便用户安装

### DISCUSSION THREAD

**Discussion by ghost** [v2fly/v2ray-core#2628] [Ideas]
v2ray目前都这是通过脚本安装和更新，不是很方便，如果创建一个ppa仓库Debian系用户就可以用apt安装和更新了，更方便。

**Comment by AkinoKaede** [comment]
V2Ray is in the Debian repository. You can install V2Ray by `apt install v2ray`
If anyone would like to maintain the PPA repository we would appreciate it.
